stratasys/machine.py

```diff
@@ -11,8 +11,7 @@
     "ktype": "6B2A268B5ED3374A",
     "prodigy": "5394D7657CED641D",
     "quantum": "76C454D532E610F7",
-    "uprint": "F3A91DBE6B0B2255",
-    "uprintse": "6B2A268B5ED3374A",
+    "uprint/SE": "F3A91DBE6B0B2255",
```

Comment:
could you rename "uprint/SE" to "uprintse"? I think it will be more friendly on Windows.